docs/meshstack.customer-group-sync.md

```diff
@@ -3,142 +3,192 @@ id: meshstack.customer-group-sync
 title: meshCustomer Group Synchronisation
 ---
 
-meshStack supports importing users and groups into meshStack from a source that supports the LDAP protocol, for example, Microsoft Active Directory. The [`Simple Paged Results Control`](https://www.ietf.org/rfc/rfc2696.txt) is used to do a paginated LDAP query to fetch the users and groups. Because of this, it is required that the LDAP server supports the `Simple Paged Results Control`. Other than importing users and groups, meshStack also has the capability to assign a [customer role](./meshcloud.customer.md#assign-meshcustomer-roles) or [partner role](./administration.index.md) to the imported group. The entities read via LDAP are imported to meshStack via the [meshObject import API](./meshstack.api.md#meshobject-api).
+meshStack supports importing users and groups into meshStack from a source that supports the LDAP protocol, for example, Microsoft Active Directory. The [`Simple Paged Results Control`](https://www.ietf.org/rfc/rfc2696.txt) is used to do a paginated LDAP query to fetch the users and groups. Because of this, it is required that the LDAP server supports the `Simple Paged Results Control`. Other than importing users and groups, meshStack also has the capability to assign a [customer role](./meshcloud.customer.md#assign-meshcustomer-roles) or [partner role](./administration.index.md) to the imported group. The entities read via LDAP are imported to meshStack via the [meshObject import API](./meshstack.api.md#meshobject-api). At the moment, we only support the import of three kinds of meshObjects: meshUser, meshGroup and meshCustomerGroupBinding. The other meshObjects, such as meshProjectUserBindings can be created via the panel.
 
 ## Configuration Reference
 
 This section describes how to configure the LDAP group synchronization in meshStack.
-For easier reference the following sections break down the configuration model in multiple parts. The union of these
-defines the full configuration model.
 
-The basic parameters required to configure the connection to LDAP are as follows.
+The synchronization process is done in three parts.
 
-<!--snippet:mesh.identityconnector.ldap.core#type-->
+* Collect
+* Transform
+* Transport
+
+### Collect
+
+In the Collect step, we connect to the LDAP server and fetch the LDAP entities. In order to do that,
+the configuration needs to contain the information required to establish a connection with the LDAP server.
+The configuration model is as follows.
+
+<!--snippet:mesh.identityconnector.ldap.SourceConfiguration#type-->
 
 
 <!--DOCUSAURUS_CODE_TABS-->
 <!--Dhall Type-->
 ```dhall
-let ConnectionConfiguration =
+let SourceConfiguration =
     {-
-      url: The LDAP connection URL which would be of the form ldap://example.com:389
+      url:
+          The LDAP connection URL, including the ldap protocol
```

Comment:
why did you get rid of the examples?

Reply:
The example is moved to it's own dhall example section below.